Cloak/data/tables/cloak-sct.tbm

```diff
@@ -351,6 +351,23 @@ end
 	
 end
 
+mn.LuaSEXPs["lua-is-cloaked"].Action = function(...) 
+    for i,data in ipairs(arg) do
+		local entry = CloakSys:GetEntryByName(data[1].Name)
+        
+        if entry == nil or entry.Status ~= 1 then
+            return false
+        end
+    end
+    return true
+end
+
+mn.LuaSEXPs["lua-player-is-cloaked"].Action = function(...) 
```

Comment:
Since this function doesn't need any arguments, just use `()` instead of `(...)`